src/main/java/org/example/internetshop/controllers/cart/AddProductToCartController.java

```diff
@@ -1,4 +1,4 @@
-package org.example.internetshop.controllers;
+package org.example.internetshop.controllers.cart;
```

Comment:
```suggestion
package org.example.internetshop.controller.cart;
```
пекеджі мають називатися в однині